twingate/internal/test/acctests/resource/resource_test.go

```diff
@@ -2542,3 +2542,79 @@ func createResourceWithBrowserOption(name, networkName, resourceName, address st
 	}
 	`, name, networkName, resourceName, address, browserOption)
 }
+
+func TestAccTwingateResourceUpdateSecurityPolicy(t *testing.T) {
+	resourceName := test.RandomResourceName()
+	theResource := acctests.TerraformResource(resourceName)
+	remoteNetworkName := test.RandomName()
+
+	policies, err := acctests.ListSecurityPolicies()
```

Comment:
I understand that correctly we neither support creating or reading ( data source ) any security policies through the provider ? 
We assume the user created these policies in the admin and copied the ID to the TF code ? 

@alexb-twingate any plans adding a datasource to Security Policies ?